tests/Unit/Evolution/Systems/ScalarWave/Test_EnergyDensity.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cmath>
+#include <cstddef>
+#include <random>
+#include <string>
```

Comment:
you don't use `cmath`

you don't use `DataBox.hpp` or `TMPL.hpp`

you won't be using many of the others after you've updated the code :)